.gitignore

```diff
@@ -26,6 +26,11 @@
 /test-write.wav
 /test-plugin-list.xml
 
+# --- CMake stuff
```

Comment:
I think you don't need to ignore this stuff. In CMake you usually create a `build/` directory and invoke `cmake` from there. Any CMake artifact will be created in the `build/` directory: just delete it to clean up the workspace.

Reply:
These files get recreated every time I do "make". I think there is something wrong with my environment, since I have no _build_ dir at all, and *.o files are mixed up with source. 
So I left these rules in place for now. Any tips how to resolve it?

Reply:
You usually create the `build/` directory, `cd` in there and invoke CMake as follows: `cmake [optional flags here] ..`. The two dots mean "upper dir".

Reply:
Okay, I think I got it. Thanks!